operate/services/manage.py

```diff
@@ -313,9 +314,10 @@ def _deploy_service_onchain(  # pylint: disable=too-many-statements,too-many-loc
 
         # TODO fix this
         os.environ["CUSTOM_CHAIN_RPC"] = ledger_config.rpc
-        os.environ[
-            "OPEN_AUTONOMY_SUBGRAPH_URL"
-        ] = "https://subgraph.autonolas.tech/subgraphs/name/autonolas-staging"
+        if "OPEN_AUTONOMY_SUBGRAPH_URL" not in os.environ:
+            os.environ[
+                "OPEN_AUTONOMY_SUBGRAPH_URL"
+            ] = "https://subgraph.autonolas.tech/subgraphs/name/autonolas-staging"
```

Comment:
I suspect whe don't need this anymore. Seems to be already defined in the framework. Please check and delete if applies.